packages/perseus/src/widgets/radio/radio.ff.tsx

```diff
@@ -20,23 +20,31 @@ type Props = WidgetProps<RenderProps, PerseusRadioRubric>;
  */
 class Radio extends RadioOld {
     ffIsOn = false;
-    radioRef = React.createRef<RadioOld | RadioNew>();
+    radioRef = React.createRef<RadioOld>();
 
```

Comment:
We only need the ref for the old component now that the focus method is removed!